core/admin/admin.php

```diff
@@ -410,71 +435,140 @@ public function elementor_dashboard_overview_widget() {
-							$recently_edited_query->the_post();
-							$document = Plugin::$instance->documents->get( get_the_ID() );
-
-							$date = date_i18n( _x( 'M jS', 'Dashboard Overview Widget Recently Date', 'elementor' ), get_the_modified_time( 'U' ) );
-							?>
-							<li class="e-overview__post">
-								<a href="<?php echo esc_attr( $document->get_edit_url() ); ?>" class="e-overview__post-link"><?php echo esc_html( get_the_title() ); ?> <span class="dashicons dashicons-edit"></span></a> <span><?php echo $date; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>, <?php the_modified_time(); ?></span>
-							</li>
-						<?php endwhile; ?>
-					</ul>
-				</div>
-			<?php endif; ?>
-			<?php if ( ! empty( $elementor_feed ) ) : ?>
-				<div class="e-overview__feed">
-					<h3 class="e-heading e-divider_bottom"><?php echo esc_html__( 'News & Updates', 'elementor' ); ?></h3>
-					<ul class="e-overview__posts">
-						<?php foreach ( $elementor_feed as $feed_item ) : ?>
-							<li class="e-overview__post">
-								<a href="<?php echo esc_url( $feed_item['url'] ); ?>" class="e-overview__post-link" target="_blank">
-									<?php if ( ! empty( $feed_item['badge'] ) ) : ?>
-										<span class="e-overview__badge"><?php echo esc_html( $feed_item['badge'] ); ?></span>
-									<?php endif; ?>
-									<?php echo esc_html( $feed_item['title'] ); ?>
-								</a>
-								<p class="e-overview__post-description"><?php echo esc_html( $feed_item['excerpt'] ); ?></p>
-							</li>
-						<?php endforeach; ?>
-					</ul>
+			<?php } ?>
+			<?php if ( ! empty( $create_new_post_type ) && $show_create_post ) { ?>
+				<div class="e-overview__create">
+					<a href="<?php echo esc_url( Plugin::$instance->documents->get_create_new_post_url( $create_new_post_type ) ); ?>" class="button"><span aria-hidden="true" class="dashicons dashicons-plus"></span> <?php echo esc_html( $create_new_label ); ?>
+					</a>
 				</div>
-			<?php endif; ?>
-			<div class="e-overview__footer e-divider_top">
-				<ul>
-					<?php foreach ( $this->get_dashboard_overview_widget_footer_actions() as $action_id => $action ) : ?>
-						<li class="e-overview__<?php echo esc_attr( $action_id ); ?>"><a href="<?php echo esc_attr( $action['link'] ); ?>" target="_blank"><?php echo esc_html( $action['title'] ); ?> <span class="screen-reader-text"><?php echo esc_html__( '(opens in a new window)', 'elementor' ); ?></span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></li>
-					<?php endforeach; ?>
+			<?php } ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Elementor dashboard widget.
+	 *
+	 * Displays the Elementor dashboard widget - recently edited section.
+	 *
+	 * Fired by `elementor_dashboard_overview_widget` function.
+	 *
+	 * @since 3.12.0
+	 * @access public
+	 *
+	 * @param array $args
+	 * @param bool $show_heading
+	 *
+	 * @return void
+	 */
+	public function elementor_dashboard_overview_recently_edited( $args = [], $show_heading = true ) {
```

Comment:
```suggestion
	public function elementor_dashboard_overview_recently_edited( array $args = [], bool $show_heading = true ) {
```
Also, maybe "is_heading_shown" or something like that?